deployments/container/Makefile

```
@@ -59,17 +57,13 @@ ifeq ($(strip $(OUT_IMAGE_VERSION)),$(DEVEL_RELEASE_IMAGE_VERSION))
 PUSH_MULTIPLE_TAGS = false
 endif
 ifeq ($(PUSH_MULTIPLE_TAGS),true)
-push-$(DEFAULT_PUSH_TARGET): push-short
+push-image: push-short
```

We shouldn't need `push-short` anymore
```suggestion
push-image:
```